deploy/k8s/falco-exporter/templates/daemonset.yaml

```diff
@@ -30,8 +30,8 @@ spec:
           imagePullPolicy: IfNotPresent
           args:
             - /usr/bin/falco-exporter
-            - --client-hostname=falco-grpc.default.svc.cluster.local
-            - --client-port=5060
+            - --client-socket=unix:///var/run/falco/falco.sock
+            
```

Comment:
Need to be fixed in the chart.

Reply:
Fixed by https://github.com/falcosecurity/falco-exporter/pull/33/commits/97390438e4082352d596117590f360096266a852